lightly/data/dataset.py

```diff
@@ -234,6 +234,8 @@ def from_torch_dataset(cls,
         # populate it with the torch dataset
         if transform is not None:
             dataset.transform = transform
+            # If dataset is a VisionDataset dataset, we need to initialize the transform
```

Comment:
Let's change this to "If dataset is a VisionDataset, we need to initialize transforms, too."